src/data/redux.test.jsx

```diff
@@ -1095,4 +1095,88 @@ describe('Data layer integration tests', () => {
       expect(state.examState.examAccessToken.exam_access_token).toBe('');
     });
   });
+
+  describe('Test checkExamEntry', () => {
+    const mockPostMessage = jest.fn();
+    const mockAddEventListener = jest.fn();
+
+    beforeEach(() => {
+      windowSpy.mockImplementation(() => ({
+        top: {
+          postMessage: mockPostMessage,
+        },
+        addEventListener: mockAddEventListener,
+        removeEventListener: jest.fn(),
+      }));
+    });
+
+    afterEach(() => {
+      jest.clearAllMocks();
+      axiosMock.reset();
+    });
+
+    it('should check application status for LTI proctored exams in a proctored state', async () => {
```

Comment:
Not sure if it's overkill, but could a test be added to check that the application status is not checked for non-LTI proctored exams in a proctored state?